llpc/context/llpcCompiler.cpp

```diff
@@ -117,6 +117,13 @@ opt<int> RelocatableShaderElfLimit("relocatable-shader-elf-limit", cl::desc("Max
                                                                    "relocatable shader ELF.  -1 means unlimited."),
                                    init(-1));
 
+// -build-shader-cache: Populates the cache by building variants of the given shaders.
+opt<bool> BuildShaderCache(
```

Comment:
as you know, LLPC use an internal cache for shsader cache, it is incompatible with pipeline binary cache,  and it can't be loaded via vkCreatePipelineCache. so what's your plan to switch to PAL pipeline binary cache for option build-shader cache?